tests/target/fn-custom-6.rs

```diff
@@ -31,7 +31,7 @@ fn bar(
 
 fn foo(a: Aaaaaaaaaaaaaa, b: Bbbbbbbbbbbbbb)
 where
-    T: UUUUUUUUUUU, {
+    T: UUUUUUUUUUU {
```

Comment:
Typically changing existing tests is not something we want to do and is usually a sign that the change [needs to be version gated](https://github.com/rust-lang/rustfmt/blob/master/Contributing.md#version-gate-formatting-changes)

Reply:
I'll do it if I have to! 👍